src/librustc_codegen_ironox/abi.rs

```diff
@@ -1,17 +1,7 @@
-// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Why are we removing the license again? If this really is based on upstream's code, I think you might have to keep it?

Reply:
Ah sorry, I just saw Laurie's comment and your response.

I've seen this too in tests, but I didn't realise they were removing all the license headers.

This raises a question. Should we have license headers on our unique code? If not, where should we put the license message? I guess there's a file somewhere?

Reply:
There's a [COPYRIGHT](https://github.com/rust-lang/rust/blob/master/COPYRIGHT) file. That might be the right place to add the license.